cellprofiler/modules/watershed.py

```diff
@@ -83,6 +85,25 @@
        Higher values result in more regularly-shaped watershed basins. 
        
        .. _compact watershed: http://scikit-image.org/docs/0.13.x/api/skimage.morphology.html#r395
+
+Selecting *Advanced Settings* will split the detected objects into smaller objects based on a seeded watershed method 
```

Comment:
Could you add a line at the end here acknowledging that this code comes from the DeclumpObjects plugin by Madison Bowden?

Reply:
Oh yes, of course! I've since added attribution to Madison w/ my most recent commit.